packages/react-generator/src/utils/formatCode.js

```
@@ -0,0 +1,198 @@
+ * 格式化代码
+ * @param {string} code 要格式化的代码
+ * @param {string} fileName 文件名
+ * @param {Object} options 自定义配置选项
+ * @returns {string} 格式化后的代码
+ */
+export function formatCode(code, fileName, options = {}) {
+  try {
+    const parser = getParser(fileName)
+    
+    if (!parser) {
+      console.warn(`[formatCode] 不支持的文件类型: ${fileName}`)
```

ESLint

[error] Unexpected console statement.

(no-console)